### ISSUE TITLE
An error occurred creating the cluster using the PCSD Web UI

### ISSUE THREAD

**Issue by Jaycean** [ClusterLabs/pcs#263]
- When I create a cluster using Web UI (PCSD), the following Error occurs: "Error At least 1 and At most 8 addresses can be specilied for a node,0 Address specified for Node1", but it is normal to create a cluster using PCS in the addresses
- pacemaker amd64 2.0.1-5
- corosync amd64 3.0.1-2
- pcs amd64 0.10.1-2
- fence-agents amd64 4.3.3-2+deb10u1
- resource-agents amd64 1:4.2.0-2
- debian

**Comment by idevat** [comment]
Can you provide information on what you filled in the form? 

Ideally, please provide also information on what was sent to the backend when a url `/manage/cluster-setup` was called. You can find it under the `Network` tab in your browsers `dev-tools` (typical shortcut for opening is `ctrl+shift+i`). I mean information that is highlighted here:

![cluster-setup-highlighted-areas](https://user-images.githubusercontent.com/16188254/96739666-5378b400-13c0-11eb-8154-b3271a35b2c0.png)

**Comment by Jaycean** [comment]
@idevat 
```
{status: "error", status_msg: null,…}
data: null
report_list: [{severity: "ERROR", code: "COROSYNC_BAD_NODE_ADDRESSES_COUNT",…},…]
0: {severity: "ERROR", code: "COROSYNC_BAD_NODE_ADDRESSES_COUNT",…}
code: "COROSYNC_BAD_NODE_ADDRESSES_COUNT"
forceable: null
info: {actual_count: 0, min_count: 1, max_count: 8, node_name: "node1", node_index: 1}
actual_count: 0
max_count: 8
min_count: 1
node_index: 1
node_name: "node1"
report_text: "At least 1 and at most 8 addresses can be specified for a node, 0 address specified for node 'node1'"
severity: "ERROR"
1: {severity: "ERROR", code: "COROSYNC_BAD_NODE_ADDRESSES_COUNT",…}
code: "COROSYNC_BAD_NODE_ADDRESSES_COUNT"
forceable: null
info: {actual_count: 0, min_count: 1, max_count: 8, node_name: "node2", node_index: 2}
actual_count: 0
max_count: 8
min_count: 1
node_index: 2
node_name: "node2"
report_text: "At least 1 and at most 8 addresses can be specified for a node, 0 address specified for node 'node2'"
severity: "ERROR"
status: "error"
status_msg: null
```

**Comment by Jaycean** [comment]
I have found the problem that node addr is not included in the sent data. I added relevant data to create it successfully, but the configuration of /etc/hosts in the previous version can also be created directly by default. I wonder why it is not available now

**Comment by idevat** [comment]
This issue has been fixed in b073db33. As a workaround, you can specify address explicitly.
